src/app/@ansyn/ol/mapSourceProviders/marco/ol-utils/mp-tile-image-source.ts

```diff
@@ -70,7 +70,7 @@ export class MpTileImageSource extends ProjectableRaster {
 			tileUrlFunction: function (tileCoord, pixelRatio, proj) {
 				const z = tileCoord[0];
 				const x = tileCoord[1];
-				const y = -tileCoord[2] - 1;
```

Comment:
What does this change mean?

Reply:
This is a change of Ido

Reply:
So, should I ask Ido? :) This is a strange change. Do you have an idea why it was made?

Reply:
it was some breaking change from 5 to 6.
[see here](https://github.com/openlayers/openlayers/blob/main/changelog/upgrade-notes.md)